src/System.Resources.ResourceManager/tests/MissingManifestResourceExceptionTests.cs

```diff
@@ -33,13 +33,5 @@ public static void ConstructorWithMessageAndInnerException()
             Assert.Equal(message, mmre.Message);
             Assert.Same(innerException, mmre.InnerException);
         }
-
-        [Fact]
-        public void Serialization()
-        {
-            const string message = "FATAL ERROR: The pizza could not be found.";
-            var ex = new MissingManifestResourceException(message);
```

Comment:
Serializatoin works, right - through the base Exceptoin type. Which probably has all the fields anyway. It just does not deserialize to the expected derived type.